### PR TITLE
Vault manager integration

### PR DESCRIPTION
**Pull request by riuvshin**

### CODE REVIEW

File: assets/schemas/vault-config/secret-engine-1.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - version
+    required:
+    - _type
+required:
+- "$schema"
+- labels
+- path_ugly
+- type
+- description
+- options
```
Comment (jmelis) [inline assets/schemas/vault-config/secret-engine-1.yml]: newline

File: assets/schema.yml [hunk 2]
```diff
@@ -1,3 +1,130 @@
...
+  - { name: log_raw, type: string, isRequired: true }
+  - { name: hmac_accessor, type: string, isRequired: true }
+  - { name: mode, type: string, isRequired: true }
+  - { name: format, type: string, isRequired: true }
+  - { name: prefix, type: string, isRequired: true }
+
+- name: VaultAudit
+  version: '1'
+  fields:
+  - { name: path_ugly, type: string, isRequired: true }
```
Comment (riuvshin) [inline assets/schema.yml]: @jmelis do you think I should `s/path_ugly/_path/g` ?

Comment (jmelis) [reply assets/schema.yml]: yes

Comment (riuvshin) [reply assets/schema.yml]: done

File: assets/schemas/vault-config/audit-1.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+          - 'true'
+          - 'false'
+        hmac_accessor:
+          type: string
+          enum:
+          - 'true'
+          - 'false'
+        mode:
+          type: string
+          pattern: '^\d+$'
```
[the remark this answers is not in the document]
Comment (riuvshin) [reply assets/schemas/vault-config/audit-1.yml]: this has tobe string

File: assets/schemas/vault-config/audit-1.yml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+          type: string
+          enum:
+          - file
+        file_path:
+          type: string
+        log_raw:
+          type: string
+          enum:
+          - 'true'
+          - 'false'
```
[the remark this answers is not in the document]
Comment (riuvshin) [reply assets/schemas/vault-config/audit-1.yml]: this has tobe string

**Pull request merged**